book_ui.rb

```diff
@@ -0,0 +1,31 @@
+class BookUserInterface
+  attr_reader :books
+
+  def initialize
+    @books = []
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+
+    print 'Author: '
+    author = gets.chomp
+
```

Comment:
- [ ] The sole responsibility of this **create_book** method should simply be to create books. The requests for user input and input validation should be handled by another method to obey the single responsibility rule of SOLID. Kindly extract these requests for inputs into another method.

  **SUGGESTION:** You can extract these requests methods into another method and have that method return and array of **title** and **author**. Then destructure the variables in **create_book** like so:

  ![elshamy-refactor](https://user-images.githubusercontent.com/36542986/157077172-38bd9e4c-705b-4817-9a04-d24a907a02c1.jpg)